firstcall.py called with 5 arguments
Fri Aug 11 15:09:46 2006
secondcall.py called with ['-c', '-x'] as arguments
5847
In syscalls.py now
